xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -589,7 +591,12 @@ bool CGUIDialogPVRTimerSettings::Save()
     }
     else if (m_endLocalTime < m_startLocalTime) // Assume the user knows what they are doing, but log a warning just in case
     {
-      CLog::Log(LOGWARNING, "Timer settings dialog: Specified recording end time < start time: expect errors!");
+      CLog::Log(LOGWARNING,
```

Comment:
What use is it to log this here? It's a user error and the user is notified about it and has to fix it before (s)he can continue.

Reply:
fair